src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/PremisesController.kt

```diff
@@ -716,7 +716,7 @@ class PremisesController(
 
     val user = usersService.getUserForRequest()
 
-    if (premises is ApprovedPremisesEntity && !user.hasAnyRole(UserRole.CAS1_MANAGER, UserRole.CAS1_MATCHER)) {
+    if (premises is ApprovedPremisesEntity && !user.hasAnyRole(UserRole.CAS1_LEGACY_MANAGER, UserRole.CAS1_MANAGER, UserRole.CAS1_MATCHER)) {
```

Comment:
It's worth noting for context that #1911 will remove this check as CAS1 will no longer be using the lost bed endpoints in the `PremisesController` by the time it's merged.